keyboards/sixkeyboard/keymaps/via/keymap.c

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Need a license header here.

You can find a sample in [`quantum/template/base/keyboard.c`](https://github.com/qmk/qmk_firmware/blob/f145696dad27893b11213b314797711541431a9f/quantum/template/base/keyboard.c).

Reply:
added license header. thank you